src/components/Text/Text.tsx

```diff
@@ -57,19 +61,33 @@ export const Text = React.forwardRef(
             color,
             whiteSpace,
             wordBreak,
+            ellipsisLines,
+            style: outerStyle,
             ...rest
         }: TextPropsWithoutRef<C>,
         ref?: TextRef<C>,
     ) => {
         const Tag: React.ElementType = as || 'span';
 
+        const style: React.CSSProperties = {
+            maxWidth: typeof ellipsis === 'number' ? ellipsis : undefined,
+            ...outerStyle,
+        };
+
         return (
             <Tag
                 ref={ref}
                 className={text(
-                    {variant, ellipsis, whiteSpace, wordBreak},
+                    {
+                        variant,
+                        ellipsis: Boolean(ellipsis),
+                        ellipsisLines,
```

Comment:
Maybe `ellipsisLines`  is better to be the `ellipsis` prop? Boolean is for normal ellipsis, Number is for line-clamp. Why we need this shortcut: `ellipsis: 200` === `style: {maxWidth: 200}`? It seems weird for me

Reply:
Very often it is necessary to limit the length of the text to no more than a fixed value. Every time add a container for this - redundant

Reply:
We can use `style: {maxWidth: <value>}`. Why put it into prop `ellipsis`, there is no relation to it

Reply:
it's not obvious to use max width with this component to work. So now it's build in